x-pack/plugins/canvas/public/state/actions/resolved_args.js

```diff
@@ -10,6 +10,7 @@ export const setLoading = createAction('setResolvedLoading');
 export const setValue = createAction('setResolvedValue');
 export const setValues = createAction('setResolvedValues');
 export const clear = createAction('clearResolvedValue');
+export const clearSome = createAction('clearResolvedValues');
```

Comment:
nit: what do you think about making this `clearValues` and the existing one `clearValue`? As far as I can tell, `clear` isn't used anywhere besides the tests, so it's not much to update and the names follow the existing pattern better.

Reply:
That's a good idea.  I'll change it.